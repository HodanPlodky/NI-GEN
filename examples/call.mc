int f() {
    return 1 + 2;
}

int main() {
    int x;
    f();
    x = f();
    return f();
}
