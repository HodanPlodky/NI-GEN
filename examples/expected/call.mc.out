input file : examples/tests/call.mc
return 3
